pkg/compare/compare.go

```diff
@@ -498,8 +502,15 @@ func (obj InfoObject) Live() runtime.Object {
 
 // Merged Returns the Injected Reference Version of the Resource
 func (obj InfoObject) Merged() (runtime.Object, error) {
+	var err error
+	if obj.allowMerge {
+		obj.injectedObjFromTemplate, err = mergeManifests(obj.injectedObjFromTemplate, obj.clusterObj)
+		if err != nil {
+			klog.Errorf("failed to properly merge the manifests for %s some diff may be incorrect: %s", apiKindNamespaceName(obj.clusterObj), err)
```

Comment:
why is the error raised with klog instead of just returning an error, i don't see an reason to treat the error different then the rest

Reply:
We can still do the diff of the un-merged manifest, my reasoning was that an overly verbose diff is better than nothing.